src/ray/core_worker/store_provider/plasma_store_provider.cc

```diff
@@ -24,10 +24,16 @@ namespace ray {
 CoreWorkerPlasmaStoreProvider::CoreWorkerPlasmaStoreProvider(
     const std::string &store_socket,
     const std::shared_ptr<raylet::RayletClient> raylet_client,
-    std::function<Status()> check_signals, std::function<void()> on_store_full)
+    std::function<Status()> check_signals, std::function<void()> on_store_full,
+    std::function<std::string()> current_call_site)
     : raylet_client_(raylet_client) {
   check_signals_ = check_signals;
   on_store_full_ = on_store_full;
+  if (current_call_site != nullptr) {
+    current_call_site_ = current_call_site;
+  } else {
+    current_call_site_ = []() { return "Error: no callsite callback"; };
```

Comment:
Do we ever expect this to be called, or is this just for testing? The message seems a bit opaque for a user.